---
title: Echovirus 11 phylogenomics
authors: Sten de Schrijver
date: "June 2024"
dataset: "https://nextstrain.org/community/stends2001/E11/PG/VP1?r=region"
abstract: "
Thanks for having a look at the preliminary results section of the phylogenomics study I have performed on Echvirus 11! Note that at this point, I've only included the complete genomes and sequences that contain a full VP1, unless specified otherwise. This amounts to 1704 sequences in total, from both Genbank (n=1402) and from clinical laboratories (n=302).
"
---

# [Echovirus 11 phylogeny VP1 - 2011](https://nextstrain.org/community/stends2001/E11/PG/VP1?dmax=2011-12-31&r=region)
One of the most recent in-depth phylogenetic analyses performed on E11 was published in 2011. By this time, the tree based on VP1 looked like this. At this point, most of the sequences had been isolated in Asia. While genotype D was the more common genotype in Europe and West Asia, about 80% of the sequences that had been isolated at this point in time in China fell into genotype A.

# [VP1 phylogeny 2011 - subgenotypes](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=Subgenotype&dmax=2011-12-31&r=region)
While in the paper that I refer to, and it other work as well, genotypes A, C and D had been subdivided further into sub-genotypes, I was not able to do this for genotype C due to the stochasticity intrinsic to the creation of these trees. Different runs rendered different trees, with different clades forming in genotype C specifically. Genotypes A and D were rather consistent on the other hand, and they show close resemblance with earlier (and published) trees. 

# [VP1 phylogeny 2011 - subgenotype D5](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=Subgenotype&dmax=2011-12-31&f_Genotype=D&r=region)
Of all sequences that were classified within genotype D, the vast majority of sequences belonged to sub-genotype D5.

# [VP1 phylogeny 2024](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=Subgenotype&r=region)
Looking at the more recent situation, we see that by far most sequences isolated the past decade and a half fall into this D5 sub-genotype, which now accounts for more sequences than all the other genotypes (and sub-genotypes) combined. It is therefore not unreasonable to look at this sub-genotype more closely, and divide this sub-genotype further, into sub-sub-genotypes.

# [VP1 phylogeny - subsubgenotypes](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=Subsubgenotype&label=Subgenotype:D5&r=region&showBranchLabels=all)
Upon closer inspection, the D5 sub-genotype can be divided in approximately 12 sub-sub-genotypes. While D5.3 and D5.7 are quite small in size, sub-sub-genotypes D5.9 and D5.12 are enormous, and harbour quite some divergence within them. Another significant portion of the sequences have not been classified into sub-sub-genotypes, either because the position of these branches are not consistent among different simulations, or because the groups in which they appear are simply difficult to classify.

# [VP1 phylogeny - subsubgenotypes over time](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=num_date&r=region&showBranchLabels=all)
This large D5.9 sub-sub-genotype is the one that emerged most recently. Note that there some sequences have been isolated recently that belong to a specific clade within the D5.12 sub-sub-genotype.

# [VP1 phylogeny - D5.9 geography](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=Subsubgenotype&label=Subsubgenotype:D5.9&r=region&showBranchLabels=all)
D5.9, the largest sub-sub-genotype, comprises sequences isolated in European countries exlusively.

# [VP1 phylogeny - D5.9 geography](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=Country&label=Subsubgenotype:D5.9&r=region&showBranchLabels=all)
These sequences have been isolated in Germany, France, the United Kingdom, Spain, Italy, Belgium and the Netherlands.

# [VP1 phylogeny - D5.12 geography](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=region&label=Subsubgenotype:D5.12&r=region&showBranchLabels=all)
Similarly to D5.9, D5.12 is quite geographically isolated, mostly isolated in China. However, the most recently isolated sequences within this sub-sub-genotyep have been found in Europe. 

# [Date of isolated sequences per region](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=num_date&l=scatter&r=region&scatterX=region&scatterY=num_date&showBranchLabels=all)
While a lot of sequences have been isolated in China, the number of sequences published has strongly reduced since the pandemic. Instead, most sequences isolated in recent years were detected in Europe, likely linked to the risen awareness created for severe Echovirus 11 infections.

# [Source of sequences](https://nextstrain.org/community/stends2001/E11/PG/VP1?c=Source&r=region&showBranchLabels=all)
This study contributes greatly to assessing the phylogenomic landscape of Echovirus 11. The sequences uploaded on Genbank barely cover this enormous genotype of D5.9, and only small part of D5.6.

# [E11 divergence over time](https://nextstrain.org/community/stends2001/E11/PG/VP1?l=clock&r=region&showBranchLabels=all)
It took some filtering of odd sequences, but the tree I showed earlier is quite consistent among different runs. The clockview therefore also seems to look quite promising!

# [Echovirus 11 Complete genomes](https://nextstrain.org/community/stends2001/E11/CG/VP1)
A little less than 400 of these sequence comprise the complete genome of the virus, that is, more than 6500 basepairs.

# [Echovirus 11 Complete genomes - subsubgenotypes](https://nextstrain.org/community/stends2001/E11/CG/VP1?c=Subsubgenotype)
Interstingly, not a single complete genome belongs to either of the sub-sub-genotypes D5.3 and D5.10. The others are represented.

# [Echovirus 11 Complete genomes - 3DPol](https://nextstrain.org/community/stends2001/E11/CG/3DPol)
Looking at 3DPol, we see another fiarly well structured tree, though genotypes E-H don't look perfect. Besides, some sequences have proven to be difficult to put into certain genotypes. Nonetheless, among multiple runs, the tree has shown remarkable stability.

# [Echovirus 11 3DPol clock view](https://nextstrain.org/community/stends2001/E11/CG/3DPol?l=clock)
The clock view looks promising too! Only a handful of outliers were identified over the past days, which have been removed since, with a good result.

# [Echovirus 11 Complete genomes - Tanglegram VP1/3DPol](https://nextstrain.org/community/stends2001/E11/CG/VP1:community/stends2001/E11/CG/3DPol)
Merging these two trees, extensive recombination is evident. Interstingly, the genotypes in VP1 link to specific clades in 3DPol as well. Let's go through some examples.

# [Echovirus 11 Tanglegram - D5.1](https://nextstrain.org/community/stends2001/E11/CG/VP1:community/stends2001/E11/CG/3DPol?c=Subsubgenotype&label=Subsubgenotype:D5.1)

# [Echovirus 11 Tanglegram - D5.4](https://nextstrain.org/community/stends2001/E11/CG/VP1:community/stends2001/E11/CG/3DPol?c=Subsubgenotype&label=Subsubgenotype:D5.4)
I may actually want to split D5.4 up into two further clades, the bottom branch is associated with a very different 3DPol sequence as well!

# [Echovirus 11 Tanglegram - D5.8](https://nextstrain.org/community/stends2001/E11/CG/VP1:community/stends2001/E11/CG/3DPol?c=Subsubgenotype&label=Subsubgenotype:D5.8)
D5.8 has a clear distinction into two genotypes of 3DPol for these sequences.

# [Echovirus 11 Tanglegram - D5.9](https://nextstrain.org/community/stends2001/E11/CG/VP1:community/stends2001/E11/CG/3DPol?c=Subsubgenotype&label=Subsubgenotype:D5.9)
Sure there are a few exceptions, but the overhwelming majority of D5.9 sequences related to a very specific 3DPol genotype (namely R).

# [Echovirus 11 Tanglegram - D5.12)(https://nextstrain.org/community/stends2001/E11/CG/VP1:community/stends2001/E11/CG/3DPol?c=Subsubgenotype&label=Subsubgenotype:D5.12)
And lastly, D5.12. Again, relating to a clear genotype in 3DPol.

# [EV-B VP1](https://nextstrain.org/community/stends2001/E11/EV-B/VP1)
Trees of all EV-B types based on VP1 are quite clear. This is not surprising perhaps, given that types are typed based on this region. 

# [EV-B tree - E11](https://nextstrain.org/community/stends2001/E11/EV-B/VP1?f_type=E11)
E11, for example, forms a clear distinct clade at the top in this tree.

# [EV-B 3DPol)(https://nextstrain.org/community/stends2001/E11/EV-B/3DPol)
However clear, and nice, the VP1 tree is, the 3DPol tree is very chaotic! Types don't show clear clades by 3DPol, instead, multiple sequence of the same type, thus containing closely related VP1 sequences, present recombinant forms in 3DPol. 

# [EV-B VP1/3DPol)([https://nextstrain.org/community/stends2001/E11/EV-B/3DPol](https://nextstrain.org/community/stends2001/E11/EV-B/VP1:community/stends2001/E11/EV-B/3DPol)
This tanglegram shows the 3DPol sequences related to each VP1 sequence, and indeed, closely related sequences in VP1 are very distinct in 3DPol.

# [EV-B VP1/3DPol - E11](https://nextstrain.org/community/stends2001/E11/EV-B/VP1:community/stends2001/E11/EV-B/3DPol?f_type=E11)
Take E11 again, for example, which seems to be related to at least 7 clearly distinct clades of 3DPol.
